docs/docs/expression_language/why.ipynb

```diff
@@ -863,7 +863,7 @@
     "\n",
     "def batch_chain(topics: list) -> list:\n",
     "    with ThreadPoolExecutor(max_workers=5) as executor:\n",
-    "        return list(executor.map(invoke_chain, topics))\n",
+    "        return list(executor.map(invoke_chain.invoke, topics))\n",
```

Comment:
(Same as above) The LCEL examples use the function invoke_chain() so this change is not correct

Reply:
Fixed: https://github.com/langchain-ai/langchain/pull/19382/commits/053b43e0d56a14df2fbb70ca9c5d4ce7fef97833